src/main/java/com/vacation_bot/core/vacation/VacationService.java

```diff
@@ -38,7 +38,7 @@ public String createVacation(final String userName, final String _startDate, fin
 
         int period = (int) ChronoUnit.DAYS.between(startDate, endDate);
 
-        UserModel user = getUserModelRepository().findByNameOrAliasesIn(userName, Arrays.asList(userName))
+        UserModel user = getUserModelRepository().findByNameOrAliasesIn(userName, Arrays.asList(userName.split(", ")))
```

Comment:
The wrong change. The client will send just one user alias. Better change `findByNameOrAliasesIn(String name, Collection<String> aliases)` to `findByNameOrAliases(String name, String alias)`.